src/ast/query.rs

```diff
@@ -749,11 +749,16 @@ impl fmt::Display for TableFactor {
             }
             TableFactor::UNNEST {
                 alias,
-                array_expr,
+                array_exprs,
                 with_offset,
                 with_offset_alias,
             } => {
-                write!(f, "UNNEST({array_expr})")?;
+                match array_exprs.len() {
+                    0 => write!(f, "UNNEST()")?,
+                    1 => write!(f, "UNNEST({})", array_exprs[0])?,
+                    _ => write!(f, "UNNEST({})", display_comma_separated(array_exprs))?,
+                }
```

Comment:
`display_comma_separated ` already handles 0,1,many. This code can be simplified like this:

```suggestion
                write!(f, "UNNEST({})", display_comma_separated(array_exprs))?;
```